datadog_checks_dev/datadog_checks/dev/tooling/commands/env/test.py

```diff
@@ -71,6 +74,8 @@ def test(ctx, checks, agent, python, dev, base, env_vars, new_env):
                 ctx.invoke(
                     start, check=check, env=env, agent=agent, python=python, dev=dev, base=base, env_vars=env_vars
                 )
+            elif env not in config_envs:
+                continue
```

Comment:
I don't quite understand what this is meant to achieve, can you please explain?

Reply:
When you run it without `--new-env`, only the configured env makes sense to run the tests into. This filters out the ones not configured.